tensorflow/jobs/docker/2.12/py3/Dockerfile.gpu

```diff
@@ -12,6 +12,16 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && apt-get clean
 
+# Ensure below libraries are updated to mitigate vulnerability
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+    openssh-client \
+    vim
```

Comment:
What `vim` version does this pull?

Reply:
Currently is it pulling Vi IMproved 8.1, but the intention here is to always pull from the latest. For openssh-client the version is OpenSSH_8.2p1 Ubuntu-4ubuntu0.5, OpenSSL 1.1.1f

Reply:
Let's also use `--only-upgrade` to not install unintentionally new packages

Reply:
The latest scan shows that these two libraries upgrades are no longer needed